### ISSUE TITLE
[Intermittent] The video's page is wrongly displayed in immersive mode when quitting and resuming the app while in immersive mode

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2906]
**[Affected Versions]:**
- Firefox Reality version 9 rc7 (b68f946)

**[Affected Platforms]:**
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened.

**[Steps to reproduce]:**
1. Navigate to a 360 video and play it in immersive mode.
2. Press the Vive button.
3. Click the Home button.
4. Navigate on other apps from the device.
5. Reopen the app and observe the behavior.

**[Expected results]:**
- The app is reopened correctly.

**[Actual results]:**
- The page of the 360 video is displayed in immersive mode.

**[Notes]:**
- I could not verify this issue on the Oculus devices because of issue #2905.
- Attached is the video recording of the issue: [link](https://drive.google.com/open?id=1uM2D74xYHr6UH3G_BfU62gLZX-HcfL5f).

**Comment by bluemarvin** [comment]
Should have been fixed by #3196 and #3198

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the latest Firefox Reality version 10-141111916 (d9a7eec). I was not able to reproduce this issue in 10 attempts an I think it's safe to close it as verified fixed an open a new one if encountered in the future since is intermittent.

Tested on HTC Vive Focus (Android 7.1.2), Oculus Quest (Android 7.1.1) and Pico Neo 2 (Android - 8.1.0) devices.
